entrypoint.sh

```diff
@@ -74,11 +74,12 @@ function deploymentcheck(){
   isnewberunning="false"
   echo "is it running ? $isnewberunning"
   count=0
-  while [ "$isnewberunning" != "true true true" ]; do
-     if [ "$count" = "50" ]; then
+  while [["$isnewberunning" == *"false"* ]]; do
+     # Give the deployment 5 minutes to be ready.
+     if [ "$count" = "300" ]; then
          exit 1;
      fi
-     sleep 30;
+     sleep 1;
```

Comment:
lets keep 5 seconds ? @andreydrozd @yashpandya-codal

Reply:
I have in mind 1 second because the implementation of the K8s deployment should have more control over how often readiness checks are run rather than an abstract GH action. Through properties such as `periodSeconds` and `initialDelaySeconds`, a K8s deployment can be configured to run readiness and liveness checks. An extra second from a GH action that checks for readiness won't affect the perception of how fast a deployment process is, but if it's running every few seconds it'll appear slower.

Reply:
@yashpandya-codal @ppatelcodal